README.md

```diff
@@ -238,6 +238,35 @@ ros2 launch stereo_image_proc stereo_image_proc.launch.py left_namespace:=bcr_bo
 
 **Warning:**  `gz-harmonic` cannot be installed alongside gazebo-classic (eg. gazebo11) since both use the `gz` command line tool.
 
+### Mapping with SLAM Toolbox
+
+SLAM Toolbox is an open-source package designed to map the environment using laser scans and odometry, generating a map for autonomous navigation.
+
+To start mapping:
```

Comment:
Modify the line to clearly mention the same the launch file works for all the supported gazebo versions.